ds3-cli-certification/src/main/java/com/spectralogic/ds3cli/certification/CertificationWriter.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2017 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3cli.certification;
+
+import ch.qos.logback.classic.Logger;
+import com.spectralogic.ds3cli.Arguments;
+import com.spectralogic.ds3cli.CommandResponse;
+import com.spectralogic.ds3cli.util.Constants;
+import com.spectralogic.ds3client.Ds3Client;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.util.Date;
+import java.util.ResourceBundle;
+
+import static com.spectralogic.ds3cli.helpers.Util.command;
+
+
+public class CertificationWriter {
+
+    private static final String EXT = ".html";
+    private static final Logger LOG =  (Logger) LoggerFactory.getLogger(CertificationWriter.class);
+
+    private final OutputStreamWriter OUT;
+    private long currentTimeTag = 0L;
+
+    public CertificationWriter(final String fileName) throws IOException {
+        OUT = new OutputStreamWriter(new FileOutputStream(fileName + EXT), "UTF-8");
+        LOG.info("NEW FILE: {}", fileName);
+        writeHeader();
```

Comment:
It should be considered a bad practice to add state changing behavior into a constructor.